liquidity-module/index.md

```diff
@@ -129,12 +129,6 @@ A relayer is software to connect two blockchains. Configure the relayer with you
 
 If you previously used the relayer, follow these steps to remove exiting relayer and Starport configurations:
 
-- Remove your existing relayer config:
```

Comment:
we don't need to remove the existing relayer config now? if you remove this step then remove the mention in line 130 too

Reply:
We have to remove the relayer config. But the relayer file exist within .starport directory. So, removing .starport removes both starport and relayer files.